docker/docker.go

```diff
@@ -11,16 +11,15 @@ const (
 	Interval = 5 * time.Second
 )
 
-type DockerClient struct {
+const SRV_NAME = "SRV_NAME"
+
+type ConcreteDockerClient struct {
```

Comment:
Strange to hear. Since we did just re-arranged the method, vars and consts order and added 2 delegator methods.